README.md

```diff
@@ -123,12 +138,15 @@ $ pyenv local <myvenv>
 Install the python packages described in the `requirements.txt`
 
 ```shell
-$ pip install -r requirements.txt
+$ cd requierements
+$ pip install -r developement.txt
 ```
 
 ### Migrate the database
 
 ```shell
+$ cd ..
+$ cd  acm_web_site
 $ python manage.py migrate

```

Comment:
This is one of the lines I referred to previously, I don't think they should be changed immediately, but taken into account when merging the second PR between this one and #66 .